spec/mongoid/enum_spec.rb

```diff
@@ -21,6 +22,23 @@ class User
       expect(klass).to have_field(field_name)
     end
 
+    it "uses prefix defined in configuration" do
+      old_field_name_prefix = Mongoid::Enum.configuration.field_name_prefix
+      Mongoid::Enum.configure do |config|
+        config.field_name_prefix = '___'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.